### PR TITLE
Add sbpl global planner

### PR DESCRIPTION
**Pull request by serraramiro1**
Added sbpl global planner.

Tested with:
`LOCALIZATION=amcl GLOBAL_PLANNER=sbpl RVIZ_CONFIG=navigation roslaunch ca_gazebo create_house.launch `

### CODE REVIEW

**Comment by eborghi10** [comment]
- [x] @eborghi10 Add planner to the Wiki

File: docker/create_ros_melodic_gazebo9/packages.txt [hunk 1]
```diff
@@ -43,4 +43,5 @@ ros-*-smach-viewer
 ros-*-teb-local-planner
 ros-*-transmission-interface
 ros-*-xacro
-ros-*-yocs-controllers
\ No newline at end of file
+ros-*-yocs-controllers
+ros-*-sbpl-lattice-planner
```
Comment (eborghi10) [inline docker/create_ros_melodic_gazebo9/packages.txt]: Move this in alphabetical order.

File: navigation/ca_move_base/launch/move_base.launch [hunk 2]
```diff
@@ -16,7 +16,7 @@
     <rosparam file="$(find ca_move_base)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
     <rosparam file="$(find ca_move_base)/config/local_costmap_params.yaml" command="load" />
     <rosparam file="$(find ca_move_base)/config/$(arg local_planner)_local_planner_params.yaml" command="load" />
-
+    <param name="SBPLLatticePlanner/primitive_filename" value="$(find ca_move_base)/config/matlab/turtle.mprim" />
```
Comment (eborghi10) [inline navigation/ca_move_base/launch/move_base.launch]: Can you move it to line 43 close to the TEB parameter?

**Review by eborghi10 — CHANGES_REQUESTED** [review]

File: docker/create_ros_melodic_gazebo9/packages.txt (hunk 1 above)
Comment (serraramiro1) [reply docker/create_ros_melodic_gazebo9/packages.txt]: Addressed

File: navigation/ca_move_base/launch/move_base.launch (hunk 2 above)
Comment (serraramiro1) [reply navigation/ca_move_base/launch/move_base.launch]: Addressed

**Review by eborghi10 — APPROVED** [review]
Screenshot from 2020-06-11 18-44-43

Cool! It's working great (besides that SBPL is too verbose) :sweat_smile: 

Good work!

**Pull request merged**